src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -12,6 +12,7 @@
       @chiRowCollapsed="e => this.rowCollapsed(e)"
       @chiRowSelected="e => this.rowSelected(e)"
       @chiRowDeselected="e => this.rowDeselected(e)"
+      @chiMobileCancel="e => this.chiMobileCancel(e)"
```

Comment:
Now I see why you need the event. You switch `showBottomNavOnMobileView` based on it but the visibility should be handler out of the box by `ChiDataTableBulkActions`. I will make the necessary changes in a separate commit.